docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+    app:
+        build:
+            context: .
+        ports:
+            - "8000:8000"
+        volumes:
+            - ./:/app/
+            - my_media:/files/media
+            - my_static:/files/static
+        command: >
+            sh -c "python manage.py makemigrations &&
+            python manage.py wait_for_db
+            && python manage.py collectstatic --noinput
+            && python manage.py migrate &&
+                      python manage.py runserver 0.0.0.0:8000"
```

Comment:
Consider using a process manager like `gunicorn` for running the Django application in production, as it is more suitable than the built-in Django development server (`runserver`).